### PR TITLE
Stage empty manifest if a staging session is given.

### PR DESCRIPTION
**Pull request by robertwb**
This allows us to run jobs on the Flink runner until we actually hook up the staging flags.

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

### CODE REVIEW

**Comment by robertwb** [comment]
R: @axelmagn

File: sdks/python/apache_beam/runners/portability/portable_runner.py [hunk 1]
```diff
@@ -104,9 +104,21 @@ def run_pipeline(self, pipeline):
     prepare_response = job_service.Prepare(
         beam_job_api_pb2.PrepareJobRequest(
             job_name='job', pipeline=proto_pipeline))
+    if prepare_response.staging_session_token:
+      # Must commit an empty manifest.
+      artifact_service = beam_artifact_api_pb2_grpc.ArtifactStagingServiceStub(
+        grpc.insecure_channel(prepare_response.artifact_staging_endpoint.url))
+      commit_manifest = artifact_service.CommitManifest(
+          beam_artifact_api_pb2.CommitManifestRequest(
```
Comment (axelmagn) [inline sdks/python/apache_beam/runners/portability/portable_runner.py]: So just to confirm, the intention is to stage an empty manifest for now until we are ready to wire in actual artifact staging in the python sdk?  If so, then I think we should add a TODO line there for clarity.

Comment (robertwb) [reply sdks/python/apache_beam/runners/portability/portable_runner.py]: Done.

**Comment by axelmagn** [comment]
LGTM once tests are green.

**Pull request merged**